Services/Auth0toAI/Auth0toAI/Startup.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Auth0toAI.Service;
+
+[assembly: FunctionsStartup(typeof(Auth0toAI.Startup))]
```

Comment:
please migrate to dotnet-isolated, using Program.cs for setup.

reason is that non-isolated is deprecated and is not going to support .NET8 later this year.